README.md

```diff
@@ -211,6 +211,49 @@ This mixin binds the url of the best fitting image to the source attribute, base
 
 This mixin binds the url of the best fitting image as the background url to the elements style attribute, based in the values provided by the `image` and `size` attribute. It also get the `responsiveImage` service injected.
 
+##Extensibility hooks
```

Comment:
this does not render correctly, because of the missing space!

Reply:
Fixed. For some reason the editor's plugin renders correctly...